src/libraries/System.Security.Cryptography.Cose/src/System/Security/Cryptography/Cose/CoseHeaderValue.cs

```diff
@@ -46,6 +64,11 @@ private static ReadOnlyMemory<byte> Encode(CborWriter writer)
             return buffer.AsMemory();
         }
 
+        /// <summary>
+        /// Creates a <see cref="CoseHeaderValue"/> instance from a signed integer."/>.
+        /// </summary>
+        /// <param name="value">The signed integer to be encoded and that the instance will represent.</param>
```

Comment:
From a black-box perspective, encoding might never happen, so the paramdoc shouldn't really bother talking about it.

```suggestion
        /// <param name="value">The value to represent.</param>
```

Reply:
If we keep "to represent" on `FromInt32`, we should keep it as well on `FromEncodedValue`.